src/main/java/ee/buerokratt/ruuter/helper/ConfigurationMappingHelper.java

```diff
@@ -1,55 +1,56 @@
-package ee.buerokratt.ruuter.service;
+package ee.buerokratt.ruuter.helper;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import ee.buerokratt.ruuter.domain.AssignStep;
 import ee.buerokratt.ruuter.domain.ConfigurationStep;
 import ee.buerokratt.ruuter.domain.HttpStep;
 import ee.buerokratt.ruuter.domain.ReturnStep;
-import ee.buerokratt.ruuter.service.exception.InvalidConfigurationException;
-import ee.buerokratt.ruuter.service.exception.InvalidConfigurationStepException;
+import ee.buerokratt.ruuter.helper.exception.InvalidConfigurationException;
```

Comment:
I'd say the previous version (`service`, not `helper`) was correct. Helpers should be common classes to create UUIDs etc. Functionalities that are not Ruuter-specific.

The same goes throughout this pull request but I only added 1 comment about it.